subiquity/server/server.py

```diff
@@ -117,6 +117,9 @@ async def client_variant_POST(self, variant: str) -> None:
             raise ValueError(f'unrecognized client variant {variant}')
         self.app.base_model.set_source_variant(variant)
 
+    async def client_variant_GET(self) -> str:
+        return self.app.base_model.client_variant
```

Comment:
I would not store it on the base_model (above), but rather on the application directly. Then, the method is used to adapt the base_model, without storing it.

Reply:
Updated.